src/UXClient/Components/GroupedBarChart/GroupedBarChart.ts

```diff
@@ -29,7 +29,7 @@ class GroupedBarChart extends ChartVisualizationComponent {
     }
 
     GroupedBarChart() { }
-    public render(data: any, options: any, aggregateExpressionOptions: any) {
+    public render(data: any, options: any, aggregateExpressionOptions?: any) {
```

Comment:
Should any README / Samples be updated for this change ?

Reply:
Since the document already mentions only one argument. It was a fix in code.